.vscode/extensions.json

```diff
@@ -2,6 +2,14 @@
   "recommendations": [
     "editorconfig.editorconfig",
     "redhat.vscode-yaml",
-    "biomejs.biome"
+    "biomejs.biome",
+    "github.remotehub",
+    "github.vscode-pull-request-github",
+    "ms-vscode.remote-repositories",
```

Comment:
I'm a little wary of this being a pre-release edition "best for VSCode insiders", but considering it's being maintained maybe less of an issue. Though again, I'm not seeing the immediate _need_ for it

Reply:
Even better, a more thorough look is that they are both based on https://github.com/microsoft/vscode-remote-repositories-github, so therefore, they seem to be the same extension. I'm going to yank them both.